cmake_modules/ParsecCompilerFlags.cmake

```diff
@@ -64,19 +64,24 @@ function(check_and_set_compiler_option)
     endif()
   endforeach()
   set(CMAKE_REQUIRED_QUIET ${save_CMAKE_REQUIRED_QUIET})
-  if( _supported_languages STREQUAL "" )
-    message(CHECK_FAIL "[none]")
-  else( _supported_languages STREQUAL "" )
-    message(CHECK_PASS "[${_supported_languages}]")
-  endif( _supported_languages STREQUAL "" )
+
+  if(NOT DEFINED ${parsec_cas_co_NAME} OR NOT ${parsec_cas_co_NAME} STREQUAL parsec_cas_co_defined_options)
+    # Something new found, output result
+    if( _supported_languages STREQUAL "" )
+      message(STATUS "Performing Test Flag ${parsec_cas_co_OPTION} supported by languages -\t[None]")
+    else( _supported_languages STREQUAL "" )
+      message(STATUS "Performing Test Flag ${parsec_cas_co_OPTION} supported by languages -\t[${_supported_languages}]")
+    endif( _supported_languages STREQUAL "" )
+  endif()
 
   if(DEFINED parsec_cas_co_COMMENT)
     set(${parsec_cas_co_NAME} "${parsec_cas_co_defined_options}" CACHE STRING "${parsec_cas_co_COMMENT}")
```

Comment:
Discussed during meeting about this not using FORCE; we will remove the STRING type and set it to INTERNAL.